### ISSUE TITLE
+manageshop toggle for abilities displayed on where/shops

### ISSUE THREAD

**Issue by Hierodule** [Arx-Game/arxcode#249]
I recently saw a complaint that the where/shops lists get cluttered, a.) because of people setting up multiple shops and b.) because every shop a player has lists all their abilities, regardless of what recipes they put up.

I thus suggest the following: 

+manageshop/addability
+manageshop/rmability

These would then filter recipes by the ability added or removed, and likewise show up on where/shops according to ability attached.

Could also have a +manageshop/addability all and /rmability all for easy access, which just adds all abilities on your sheet.

Cheers and thanks for the consideration.

**Comment by rozzingit** [comment]
Yeah, the where/shops command can be pretty unhelpful. I like the solution proposed in #203 the best because it doesn't require any further input from the player; it would just have where/shops base its output on the recipes made available in a given shop.

**Comment by Hierodule** [comment]
That also works well :)
